src/js/components/SelectMultiple/SelectMultipleContainer.js

```diff
@@ -0,0 +1,614 @@
+                plain
+              />
+            </Box>
+          ) : (
+            <Box
+              pad="xsmall"
+              direction="row"
+              justify="between"
+              gap="small"
+              flex={false}
+            >
+              <SelectionSummary
+                allOptions={allOptions}
+                clearRef={clearRef}
+                disabled={disabled}
+                disabledKey={disabledKey}
+                isSelected={isSelected}
+                limit={limit}
+                onChange={onChange}
+                options={options}
+                search={search}
+                setActiveIndex={setActiveIndex}
+                value={value}
+                valueKey={valueKey}
+              />
+            </Box>
+          )}
+
+          {onSearch && (
+            <Box pad={!customSearchInput ? 'xsmall' : undefined} flex={false}>
+              <Keyboard
+                onEnter={(event) => {
+                  onNextOption(event);
+                }}
+              >
+                <SelectTextInput
+                  a11yTitle="Search to filter options."
+                  focusIndicator={!customSearchInput}
+                  size="small"
+                  ref={searchRef}
+                  type="search"
+                  value={search || ''}
+                  placeholder={searchPlaceholder}
+                  onChange={(event) => {
+                    const nextSearch = event.target.value;
+                    setSearch(nextSearch);
+                    setActiveIndex(-1);
+                    onSearch(nextSearch);
+                  }}
+                />
+              </Keyboard>
+            </Box>
+          )}
+          <Box flex={false}>{help}</Box>
+          <OptionsBox
+            role="listbox"
+            tabIndex="0"
+            ref={optionsRef}
+            aria-multiselectable
+            onMouseMove={() => setKeyboardNavigation(false)}
+            aria-activedescendant={optionsRef?.current?.children[activeIndex]}
+          >
+            {options.length > 0 ? (
+              <InfiniteScroll
+                items={options}
+                step={theme.select.step}
+                onMore={onMore}
+                replace={replace}
+                show={activeIndex !== -1 ? activeIndex : undefined}
+              >
+                {(option, index, optionRef) => {
+                  const iValue =
+                    valueKey && valueKey.reduce
+                      ? applyKey(option, valueKey)
+                      : option;
+                  const optionDisabled = checkDisabled(
+                    index,
+                    disabled,
+                    disabledKey,
+                    options,
+                    valueKey || labelKey,
+                  );
+                  const optionSelected = value.includes(iValue);
+                  const optionActive = activeIndex === index;
+                  const iLabel = getOptionLabel(
```

Comment:
```suggestion
                  const optionLabel = getOptionLabel(
```
feel like this is a little more aligned w grommet conventions

Reply:
Yes, fixed in latest commit